scripts/run_testnet.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env sh
+
+# Function to kill background processes
+cleanup() {
+    echo "Stopping background processes and removing files..."
+    find . -type f -name "validator_output.txt" -delete
+    find . -type f -name "node_output.txt" -delete
+    kill "$validator_pid" "$node_pid" 2>/dev/null
+}
+
+# Trap EXIT signal to ensure cleanup runs on script exit
+trap cleanup EXIT
+
+# build release
+cargo build --release
```

Comment:
In the CI there is already a build binary cached. You should use this one and no rebuild everything

Reply:
I will keep the script as is for local testing but re-write the ci yml file instead.